client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-placeholder-dialog.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+
+class PaymentMethodStripePlaceholderDialog extends Component {
+	render() {
+		const { translate } = this.props;
+		const buttons = [
+			{ action: 'cancel', disabled: true, label: translate( 'Cancel' ), onClick: noop },
```

Comment:
Why do we need such button?

Reply:
We don't technically, but I thought it made the transition to the non-placeholder dialog look a little less jarring.